src/main/java/uk/nhs/digital/nhsconnect/lab/results/inbound/queue/FhirDataToSend.java

```diff
@@ -2,16 +2,18 @@
 
 import lombok.Getter;
 import lombok.Setter;
-import org.hl7.fhir.dstu3.model.Parameters;
+import org.hl7.fhir.dstu3.model.Bundle;
 
 @Getter
 @Setter
 public class FhirDataToSend {
-    private Object content;
+
+    private Bundle content;
     private String operationId;
 
-    public FhirDataToSend setContent(final Parameters parameters) {
-        this.content = parameters;
+    public FhirDataToSend setContent(final Bundle bundle) {
```

Comment:
I think this setter is redundant as lombok's `@Setter` annotation generates the same thing

Reply:
Good point, didn't notice that when I was updating it. Yeah I can remove that